demo/index.html

```diff
@@ -86,7 +86,8 @@
     }
 
     core-tooltip.fancy:hover::shadow .core-tooltip,
-    core-tooltip.fancy:focus::shadow .core-tooltip {
+    core-tooltip.fancy:focus::shadow .core-tooltip,
+    core-tooltip.fancy[show]::shadow .core-tooltip {
```

Comment:
This is a bugfix for the demo script of [Toggle all] button that is unrelated to the dependency removal. I'll exclude the unrelated changes 15ec8de, dff0c7f if you prefer to merge a single-purpose patch.